cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/PostgresSinkConsumerIT.java

```diff
@@ -0,0 +1,277 @@
+package com.yugabyte.cdcsdk.testing;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.lang.Double;
+import java.lang.Integer;
+import java.lang.String;
+import java.net.InetAddress;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Iterator;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import org.apache.kafka.clients.consumer.*;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.YugabyteYSQLContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.debezium.testing.testcontainers.*;
+
+public class PostgresSinkConsumerIT {
+    private static final Logger LOGGER = LoggerFactory.getLogger(PostgresSinkConsumerIT.class);
+    private static String HOST_ADDRESS = "127.0.0.1";
+    protected static String BOOTSTRAP_SERVER = "127.0.0.1:9092";
+    private KafkaConsumer<String, JsonNode> consumer;
+    private static List<Map<String, Object>> expected_data = new ArrayList<>();
+    private static int recordsInserted = 5;
+    protected static KafkaContainer kafka;
+    private static YugabyteYSQLContainer ybContainer;
+    private static GenericContainer<?> cdcContainer;
+    private static final DockerImageName KAFKA_TEST_IMAGE = DockerImageName.parse("confluentinc/cp-kafka:6.2.1");
+    private static Network network;
+    private static DockerImageName myImage = DockerImageName.parse("debezium/example-postgres:1.6").asCompatibleSubstituteFor("postgres");
+    private static PostgreSQLContainer<?> postgreSQLContainer;
+    private static DebeziumContainer debeziumContainer;
+    private static String POSTGRES_IP;
```

Comment:
Since all these variables are going to be there in this class only, you should not make them`static`.